corehq/apps/userreports/tests/test_choice_provider.py

```diff
@@ -553,6 +554,13 @@ def setUpClass(cls):
         choices.sort(key=lambda choice: choice.display)
         cls.choice_provider = DomainChoiceProvider(cls.report, None)
         cls.static_choice_provider = StaticChoiceProvider(choices)
+        role = UserRole.create(
+            domain=cls.domain_a,
+            name='Can View Registry Data',
+            permissions=Permissions(view_data_registry_contents=True),
+        )
+        cls.web_user.set_role(cls.domain_a, role.get_qualified_id())
+        cls.web_user.save()
```

Comment:
don't you need to add the user to the query contexts that get passed in to the `.query` method?